Source/SuperMario3DWorld/rules.txt

```diff
@@ -5,7 +5,12 @@ path = "Super Mario 3D World/Graphics/Resolution"
 description = Changes the resolution of the game. Known issues: Depth of field isn't upscaled yet, "zoom-transitions" are also not upscaled yet.
```

Comment:
Since you removed depth rules, wouldn't it make sense to remove the mention of it from here?

Reply:
Removed